liveview-android/src/main/java/org/phoenixframework/liveview/data/dto/RadioButtonDTO.kt

```diff
@@ -0,0 +1,147 @@
+package org.phoenixframework.liveview.data.dto
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.material3.RadioButton
+import androidx.compose.material3.RadioButtonColors
+import androidx.compose.material3.RadioButtonDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import kotlinx.collections.immutable.ImmutableMap
+import kotlinx.collections.immutable.toImmutableMap
+import org.phoenixframework.liveview.data.core.CoreAttribute
+import org.phoenixframework.liveview.data.mappers.JsonParser
+import org.phoenixframework.liveview.domain.base.ComposableViewFactory
+import org.phoenixframework.liveview.domain.base.PushEvent
+import org.phoenixframework.liveview.domain.extensions.privateField
+import org.phoenixframework.liveview.domain.extensions.toColor
+import org.phoenixframework.liveview.domain.factory.ComposableTreeNode
+
+/**
+ * Material Design radio button.
+ * ```
+ * <Row verticalAlignment="center">
+ *   <RadioButton value="A" phx-change="setRadioOption" selected={"#{@radioOption == "A"}"} />
```

Comment:
Boolean attributes in HEEx can be written like this:
```heex
<RadioButton selected={@radioOption == "A"} />
```
This will automatically add/remove the `selected` attribute from the markup.

In HTML, [boolean attributes](https://developer.mozilla.org/en-US/docs/Glossary/Boolean/HTML) are detected by presence, not value. So even if `selected="false"`, the `<RadioButton>` should still be selected.

Reply:
Got it. Please, open a separate issue for that. I have to refactor all boolean properties to follow this approach...